docs/installation.rst

```diff
@@ -434,8 +434,37 @@ The connection string for BigQuery looks like this ::
 
     bigquery://{project_id}
 
+Additionally, you will need to configure authentication via a Service Account. Create your Service Account via the
```

Comment:
NIT: I think for `.md` or `.rst` people tend to break line at either 80 or 100. I'm fine with either personally. I'm not sure if it's worth enforcing or not